app/views/users/show.html.erb

```diff
@@ -3,6 +3,7 @@
 
   <%= button_to 'Log Out', logout_path, method: 'delete', class: "btn btn-primary mb1 bg-teal" %>
   <%= button_to 'Connect to Github', connect_path, class: "btn btn-primary mb1 bg-teal" unless current_user.github_token%>
+  <%= button_to 'Send an Invite', invite_path, method: :get, class: "btn btn-primary mb1 bg-teal" if current_user.github_token%>
```

Comment:
Little ambiguous. Maybe the button says "Invite a Friend" or something....

Reply:
#6 user story refers to it as "Send an Invite". I mean, I'm sure we can ask to have more flexibility with the wording. Or perhaps in the headers of the form tag we could add more information.